docs/source/about/contact.rst

```diff
@@ -0,0 +1,16 @@
+Contact us
+=====================================
+
+**MetaSynth** is a project by the `ODISSEI Social Data Science
+(SoDa) <https://odissei-data.nl/nl/soda/>`__ team. 
+
+Do you have
```

Comment:
It would be nice to show a bit more that we welcome PRs as well, so they can also contribute.

Reply:
Agreed, the next commit includes a guide on how to contribute and has a few links around the docs that encourage PR's and direct the reader over there, including one on this page!